components/dashboard/src/data/setup.tsx

```diff
@@ -16,6 +16,11 @@ import { Message } from "@bufbuild/protobuf";
 import { ReactQueryDevtools } from "@tanstack/react-query-devtools";
 import { FunctionComponent } from "react";
 import debounce from "lodash.debounce";
+// Need to import all the protobuf classes we want to support for hydration
```

Comment:
I swapped these to imports for consistency and I think it makes it a bit clearer what we're importing and why. It seems to work fine this way, not sure if there was another reason you had used require's @svenefftinge? 

Normally I think imports are more desirable as they allow for static analysis (and allow for tree-shaking unused exports). In this case we're importing everything anyways, so that benefit is probably moot, but I think the consistency of imports at the beginning of the module is still valuable for readability. Happy to adjust back if we don't want that though.